runtime/gc_check/CheckBase.hpp

```diff
@@ -101,6 +101,7 @@
 #define J9MODRON_GCCHK_MISC_ALWAYS_DUMP_STACK ((UDATA)0x00004000)
 #define J9MODRON_GCCHK_MISC_DARKMATTER ((UDATA)0x00008000)
 #define J9MODRON_GCCHK_MISC_MIDSCAVENGE ((UDATA)0x00010000)
+#define J9MODRON_GCCHK_VALID_INDEXABLE_DATA_ADDRESS ((UDATA)0x00020000)
```

Comment:
These definitions should be exposed to DDR to avoid repeating the definitions in java code.

Reply:
I believe this is now resolved based on the discussion and comment https://github.com/eclipse-openj9/openj9/pull/12910/files#r676847957

Reply:
I don't agree. I'll resolve this and make new comments where I think they're needed.